components/Modals/AddProjectModal/schema.ts

```diff
@@ -9,5 +9,16 @@ export const formSchema = z.object({
     }),
   tags: z.string().min(1).max(160),
   description: z.string().min(160).max(500),
-  liveUrl: z.string().optional(),
+  liveUrl: z
+    .string()
+    .refine(
+      (value) =>
+        !value ||
+        new RegExp(
+          '^https?://(www\\.)?[a-zA-Z0-9_-]+(\\.[a-zA-Z]{2,})+(/[a-zA-Z0-9_-]*)*$'
+        ).test(value),
+      {
+        message: 'Invalid URL (We support only live websites now)',
```

Comment:
everything looks fine, just one change - could you make the `https` as mandatory? currently `http` passed the test
<img width="541" alt="Screenshot 2023-12-20 at 7 12 48 PM" src="https://github.com/rohitdasu/projectmate/assets/48400770/448ca01b-c329-4632-a957-3f68205a2b03">